development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
```

Comment:
Both QGIS 2/3, but these docs should not cover building older 2.x branches. Qt Creator 4.x is very different from the 3.x series, especially with regards to CMake support. I recommend using 4.2+, as the CMake support is even better. Newer versions are slated to offer even more CMake integration, so update often.